ts/screens/profile/FiscalCodeScreen.tsx

```diff
@@ -52,14 +49,18 @@ const styles = StyleSheet.create({
   }
 });
 
-class FiscalCodeScreen extends React.PureComponent<Props, State> {
-  constructor(props: Props) {
-    super(props);
-    this.state = {
-      showAsLandscape: false,
-      showBackSide: false
-    };
-  }
+class FiscalCodeScreen extends React.PureComponent<Props> {
+  private showModal = () => {
```

Comment:
instead using _// tslint:disable-next-line:no-unused-expression_ could this code be written as follow?
```suggestion
 private showModal = () => {
    if (this.props.profile !== undefined) {
      this.props.showModal(
        <FiscalCodeLandscapeOverlay
          onCancel={this.props.hideModal}
          profile={this.props.profile}
        />
      );
    }
  };
```

Reply:
updated within https://github.com/teamdigitale/io-app/pull/1228/commits/0dc2755797e6953ba75d1ed3fe8604c3ef5ba966